src/post/content.ts

```diff
@@ -207,6 +212,44 @@ export class ContentPreparer {
         return new Set(mentions);
     }
 
+    /**
+     * Updates existing hyperlinks with data-profile attribute and rel attributes
+     */
+    private updateMentions(content: string, mentions: Mention[]) {
+        try {
+            const html = parse(content);
+            const links = html.querySelectorAll('a');
+
+            for (const mention of mentions) {
+                const apId = mention.account.apId
+                    .toString()
+                    .replace(/\/+$/, '');
+                const url = mention.account.url.toString().replace(/\/+$/, '');
+
+                // Find all links that matches the mentioned account
+                for (const link of links) {
```

Comment:
We've got a nested loop here, which we could probably flatten - but honestly I think it's fine for now - I don't see us dealing with a number of a tags or mentions that's going to be an issue